samples/snippets/storage_download_into_memory.py

```diff
@@ -37,7 +37,7 @@ def download_blob_into_memory(bucket_name, blob_name):
     # any content from Google Cloud Storage. As we don't need additional data,
     # using `Bucket.blob` is preferred here.
     blob = bucket.blob(blob_name)
-    contents = blob.download_as_string()
+    contents = blob.download_as_bytes()
 
     print(
         "Downloaded storage object {} from bucket {} as the following string: {}.".format(
```

Comment:
Let's update this to "the following bytestring" or "the following bytes object".

We need to verify this prints correctly even when the bytestring is non-ascii. The test should include an example where that is the case. Maybe we should also decode the bytestring before it is printed, assuming it is utf-8?

Reply:
Good catch; updated sample with decoding